schema/me/consignments/submission.js

```diff
@@ -0,0 +1,163 @@
+      value: "Mixed Media",
+    },
+    PERFORMANCE_ART: {
+      value: "Performance Art",
+    },
+    INSTALLATION: {
+      value: "Installation",
+    },
+    VIDEO_FILM_ANIMATION: {
+      value: "Video/Film/Animation",
+    },
+    ARCHITECTURE: {
+      value: "Architecture",
+    },
+    FASHION_DESIGN_AND_WEARABLE_ART: {
+      value: "Fashion Design and Wearable Art",
+    },
+    JEWELRY: {
+      value: "Jewelry",
+    },
+    DESIGN_DECORATIVE_ART: {
+      value: "Design/Decorative Art",
+    },
+    TEXTILE_ARTS: {
+      value: "Textile Arts",
+    },
+    OTHER: {
+      value: "Other",
+    },
+  },
+})
+
+export const SubmissionType = new GraphQLObjectType({
+  name: "Submission",
+  description: "A work to be consigned to the user",
+  fields: {
+    id: {
+      description: "Convection id.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    additional_info: {
+      description: "The type of participant, e.g. Partner or User",
+      type: GraphQLString,
+    },
+    artist_id: {
+      decription: "The gravity ID for an Artist",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    authenticity_certificate: {
+      description: "Does the artwork come with an certificate of authenticity?",
+      type: GraphQLBoolean,
+    },
+    category: {
+      description: "The set in which to put the work",
+      type: SubmissionCategoryAggregation,
+    },
+    deadline_to_sell: {
+      description: "If there is a hard date the work needs to be sold by",
+      type: GraphQLString,
+    },
+    depth: {
+      description: "The depth of the work",
+      type: GraphQLString,
+    },
+    dimensions_metric: {
+      description: "A string, either CM or IN",
+      type: SubmissionDimensionAggregation,
+    },
+    edition: {
+      description: "The version of individual work if from a set",
+      type: GraphQLString,
+    },
+    edition_number: {
+      description: "The number of the individual work if in a set",
+      type: GraphQLString,
+    },
+    edition_size: {
+      description: "The whole size of the set of works",
+      type: GraphQLString,
+    },
+    height: {
+      description: "The height of the work",
+      type: GraphQLString,
+    },
+    location_city: {
+      description: "The city where the work currently resides",
+      type: GraphQLString,
+    },
+    location_country: {
+      description: "The country where the work currently resides",
+      type: GraphQLString,
+    },
+    location_state: {
+      description: "The state where the work currently resides",
+      type: GraphQLString,
+    },
+    medium: {
+      description: "The materials in which the work is created",
+      type: GraphQLString,
+    },
+    provenance: {
+      description: "The history of an work",
+      type: GraphQLString,
+    },
+    signature: {
+      description: "Is this work signed?",
+      type: GraphQLBoolean,
+    },
+    state: {
+      description: "A description of the work's state",
```

Comment:
I'm not sure if we need this here-- this is used internally in convection right now to track how far along in the consignments process a submission is, but won't be inputted by users.